exporters/prometheus/include/opentelemetry/exporters/prometheus/exporter_utils.h

```diff
@@ -0,0 +1,136 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+#include "opentelemetry/sdk/metrics/aggregation/aggregation.h"
+#include "opentelemetry/sdk/metrics/instruments.h"
+#ifndef ENABLE_METRICS_PREVIEW
+
+#  include <string>
+#  include <vector>
+
+#  include "opentelemetry/metrics/provider.h"
+#  include "opentelemetry/sdk/metrics/aggregation/default_aggregation.h"
+#  include "opentelemetry/sdk/metrics/aggregation/histogram_aggregation.h"
```

Comment:
Also, this one, do we need it, as it was also supposed to be internal to SDK? We may have to change the SDK design if this is necessary here.

Reply:
thanks, removed.